app/models/service_offering.rb

```diff
@@ -13,4 +14,8 @@ class ServiceOffering < ApplicationRecord
 
   acts_as_tenant(:tenant)
   acts_as_taggable_on
+
+  def icon_data
+    service_offering_icon&.data
+  end
```

Comment:
We're trying to stay away from having methods in the models as much as possible to keep them clean, it is just as easy for the caller to do this.

Reply:
right, so this is similar to tags, so faking an attribute, so we can use it in API response

alternative here is to just expose service_offering_icon_id and service_offering_icons collection?

Reply:
Ah hm, how about we start with the API matching the modeling then change it if we need to?

Reply:
ok (my thinking was starting with what is most convenient for the user)